lib/SILOptimizer/IPO/DeadFunctionElimination.cpp

```diff
@@ -31,6 +31,11 @@ STATISTIC(NumDeadGlobals, "Number of dead global variables eliminated");
 
 namespace {
 
+static bool loweredFunctionHasGenericArguments(const SILFunction *f) {
```

Comment:
We could add this as a SILFunction member function, e.g. `SILFunction::isGeneric()`

Reply:
👍